app/controllers/api/v1/mixins/processor.rb

```diff
@@ -0,0 +1,11 @@
+module Api
+  module V1
+    module Mixins
+      module Processor
+        def process_service(class_name, args, namespace: "Catalog")
+          "V1x0::#{namespace}::#{class_name}".constantize.new(*args).process
```

Comment:
* Note: We call `.process` here which will clean up the controllers and force us to keep our current procedure of exposing `.process` on our services.